plugins/woocommerce/includes/wc-order-functions.php

```diff
@@ -1148,5 +1148,18 @@ function wc_create_order_note( $order_id, $note, $is_customer_note = false, $add
  * @return bool         True on success, false on failure.
  */
 function wc_delete_order_note( $note_id ) {
-	return wp_delete_comment( $note_id, true );
+	$delete = wp_delete_comment( $note_id, true );
+
+	if($delete) {
```

Comment:
Spacing here is incorrect. 
```suggestion
	if ( $delete ) {
```